### ISSUE TITLE
Forms Previewer does not always show the Content in ContentView with ControlTemplate and ContentPresenter

### ISSUE THREAD

**Issue by dimitarcnbs** [xamarin/Xamarin.Forms#5798]
### Description
Forms Previewer does not always show the Content in ContentView with ControlTemplate and ContentPresenter.

### Expected Behavior
Content should always show regardless if defined before or after the ControlTemplate is defined.

### Actual Behavior
It depends on where the content is defined in the XAML. 
If Content is defined **before** the ControlTemplate then the Content **does show**.
If Content is defined **after** the ControlTemplate then the Content **doesn't show**.

### Example code
```
<?xml version="1.0" encoding="UTF-8"?>
<ContentView 
    xmlns="http://xamarin.com/schemas/2014/forms" 
    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
    x:Class="App1.View1"
    >
    <StackLayout>
        <ContentView>

            <!-- working -->
            <ContentView.Content>
                <Label Text="content text"/>
            </ContentView.Content>

            <ContentView.ControlTemplate>
                <ControlTemplate>
                    <StackLayout >
                        <Label Text="control template text"/>
                        <ContentPresenter/>
                    </StackLayout >
                </ControlTemplate>
            </ContentView.ControlTemplate>

            <!-- not working -->
            <!-- <ContentView.Content> -->
            <!--     <Label Text="content text"/> -->
            <!-- </ContentView.Content> -->

        </ContentView>
    </StackLayout>
</ContentView>
```
### Screenshots

Working:
![Annotation 2019-04-03 101554](https://user-images.githubusercontent.com/47161856/55486008-9108c700-55f9-11e9-8987-97fb4c8d0518.png)

Not Working:
![Annotation 2019-04-03 101921](https://user-images.githubusercontent.com/47161856/55486271-06749780-55fa-11e9-8612-892cde17353f.png)

### Basic Information

- Version with issue: Xamarin.Forms 3.6.0.264807
- IDE: Visual Studio 2017
- Platform Target Frameworks:
  - iOS:  12.1

> VS bug [#837773](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/837773)

**Comment by samhouts** [comment]
[Issue5798.zip](https://github.com/xamarin/Xamarin.Forms/files/3075531/Issue5798.zip)

**Comment by samhouts** [comment]
closed by #7044
